doc/io.rst

```diff
@@ -970,12 +970,12 @@ with consolidated metadata. To write consolidated metadata, pass the
 ``consolidated=True`` option to the
 :py:attr:`Dataset.to_zarr` method::
 
-    ds.to_zarr('foo.zarr', consolidated=True)
+    ds.to_zarr("foo.zarr", consolidated=True)
 
-To read a consolidated store, pass the ``consolidated=True`` option to
-:py:func:`open_zarr`::
+To read a consolidated store, pass ``{"consolidated": True}`` to the
+backend_kwargs option when using :py:func:`open_dataset` with ``engine="zarr"``::
 
-    ds = xr.open_zarr('foo.zarr', consolidated=True)
+    ds = xr.open_dataset("foo.zarr", engine="zarr", backend_kwargs={"consolidated": True})
```

Comment:
I appreciate that it's nice to be able to use `open_dataset` and `open_mfdataset` with zarr, but these examples really seem a bit less readable to me! This feels like an argument in favor of keeping around `open_zarr`, as a version of `open_dataset` with better defaults (and keyword arguments) for zarr. Potentially it would eventually make sense to have dedicated `open_*` functions for each file-format xarray supports?

Reply:
Yep, I can revert the changes to `io.rst` here (i.e. keep `open_zarr` as the documented way of opening Zarr files instead of using `open_dataset`).